### PR TITLE
Inventorio support

### PR DESCRIPTION
**Pull request by ruvaldak**
Once again, this needs to be able to be configurable, or autodetect when Inventorio is installed (and it's inventory replacement is toggled, since you can switch back and forth between the inventorio inventory, and vanilla inventory)

### CODE REVIEW

File: src/main/java/net/marshmallow/BetterRecipeBook/Mixins/Centered/MoveEffects.java [hunk 1]
```diff
@@ -32,7 +32,7 @@ private int moveEffectsRight(int i) {
             if (MinecraftClient.getInstance().player.isCreative()) {
                 return i + 323;
             } else {
-                return i + 304;
+                return i + 323;
```
Comment (mrshmllow) [inline src/main/java/net/marshmallow/BetterRecipeBook/Mixins/Centered/MoveEffects.java:35]: Why are we doing this?

Comment (ruvaldak) [reply src/main/java/net/marshmallow/BetterRecipeBook/Mixins/Centered/MoveEffects.java:35]: Admittedly, it was a lazy number change. With Inventorio installed, the width of the inventorio inventory is exactly the same as that of the Creative inventory.

Comment (ruvaldak) [reply src/main/java/net/marshmallow/BetterRecipeBook/Mixins/Centered/MoveEffects.java:35]: Same goes for the change in RemoveRecipeBookOffset. The book widget touches the edge of the inventorio which it should be slightly offset. Without the change here in MoveEffects, potion effects overlap the inventorio GUI

**Comment by ruvaldak** [comment]
> hmm, there are a lot of problems with inventorio. The button moves around. I think its better to be fixed on the inventorio side.

I'll mess around with it, see what I can do. You might be right - Inventorio is rather janky at times.

**Pull request closed without merging**